src/Microsoft.ML.Data/Transforms/ExtensionsCatalog.cs

```diff
@@ -70,11 +70,27 @@ public static ColumnSelectingEstimator DropColumns(this TransformsCatalog catalo
             => ColumnSelectingEstimator.DropColumns(CatalogUtils.GetEnvironment(catalog), columnsToDrop);
 
         /// <summary>
-        /// ColumnSelectingEstimator is used to select a list of columns that user wants to drop from a given input.
+        /// ColumnSelectingEstimator is used to select a list of columns that user wants to keep from a given input.
         /// </summary>
+        /// <remarks>
+        /// <format type="text/markdown">
+        /// <see cref="SelectColumns"/> operates on the schema of an input IDataView,
+        /// either dropping unselected columns from the schema or keeping them but hiding them from the user. Keeping columns hidden
+        /// is recommended when it is necessary to understand how the inputs of a pipeline map to outputs of the pipeline. This feature
+        /// is used, for example, in saving models so that the model can be easily debugged at a later time. For more information,
+        /// please refer to [IDataView Design Principles](~/../docs/samples/docs/code/IDataViewDesignPrinciples.md).
+        /// </format>
+        /// </remarks>
         /// <param name="catalog">The transform's catalog.</param>
         /// <param name="keepColumns">The array of column names to keep.</param>
         /// <param name="keepHidden">If true will keep hidden columns and false will remove hidden columns.</param>
+        /// <example>
+        /// <format type="text/markdown">
+        /// <![CDATA[
+        /// [!code-csharp[Concat](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/SelectColumns.cs)]
```

Comment:
>Concat [](start = 26, length = 6)

SelectColumns

Reply:
Not related to your changes, but I think this should be SelectColumn (without the s) to keep inline with the Estimator name ColumnSelectingEstimator.

---
In reply to: [253242036](https://github.com/dotnet/machinelearning/pull/2380#discussion_r253242036) [](ancestors = 253242036)

Reply:
Well, the API is for a list of columns, so the wording matches the use here.